project/LinkerdBuild.scala

```diff
@@ -323,9 +339,43 @@ object LinkerdBuild extends Base {
       .dependsOn(core, admin, configCore)
       .withBuildProperties()
 
+    /**
+     * An assembly-running script that adds the namerd plugin directory
+     * to the classpath if it exists.
+     */
+    val execScript =
+      """|#!/bin/sh
+         |
+         |jars="$0"
+         |if [ -n "$N4D_HOME" ] && [ -d $N4D_HOME/plugins ]; then
+         |  for jar in $N4D_HOME/plugins/*.jar ; do
+         |    jars="$jars:$jar"
+         |  done
+         |fi
+         |DEFAULT_JVM_OPTIONS="-Xms40M -Xmx40M                       \
+         |   -Djava.net.preferIPv4Stack=true       \
+         |   -Dsun.net.inetaddr.ttl=60             \
+         |   -XX:+AggressiveOpts                   \
+         |   -XX:+UseConcMarkSweepGC               \
+         |   -XX:+CMSParallelRemarkEnabled         \
+         |   -XX:+CMSClassUnloadingEnabled         \
+         |   -XX:+ScavengeBeforeFullGC             \
+         |   -XX:+CMSScavengeBeforeRemark          \
+         |   -XX:+UseCMSInitiatingOccupancyOnly    \
+         |   -XX:CMSInitiatingOccupancyFraction=70 \
+         |   -XX:ReservedCodeCacheSize=32m         \
+         |   -XX:+CMSClassUnloadingEnabled         \
```

Comment:
oh, i'm saying that ` -XX:+CMSClassUnloadingEnabled` is specified twice ;)

I opted to declare flags separately for namerd and linkerd because they probably _should_ diverge...

Also, we may not want to set -Xms/mx here -- or if we do we  should support easy override?